<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAccountPage</class>
 <widget class="QDialog" name="AddAccountPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="LoginPassword">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginToAccBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="Isfound_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="LoginUsername">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
